sleap/config/labeled_clip_form.yaml

```diff
@@ -18,6 +18,14 @@ main:
     label: Use GUI Visual Settings (colors, line widths)
     type: bool
     default: true
+  - name: show_og_video
+    label: Show Original Video in Background
+    type: bool
+    default: true
+  - name: background_color
+    label: Background Color
+    type: list
+    options: Black,White,Grey
```

Comment:
Since we use two parameters to set the same thing, maybe we can combine these into one `type: list` and have the default be the video as the background?